samples/excel/85-preview-apis/data-types-web-image.yaml

```
@@ -0,0 +1,206 @@
+            // This method inserts a web image into the currently selected cell.
+
+            // Retrieve image data from the input fields and then clear the input fields.
+            const imageUrl = $("#url").val() as string;
+            const imageAltText = $("#alt-text").val() as string;
+            clearForm();
+
+            // Load the active cell.
+            const activeCell = context.workbook.getActiveCell();
+            activeCell.load();
+            await context.sync();
+
+            if (!imageUrl) {
+              console.log("Please enter an image URL.");
+              return;
+            }
+
+            // Insert web image into the active cell.
+            activeCell.valuesAsJson = [
+              [
+                {
+                  type: "WebImage",
+                  basicType: "Error",
+                  basicValue: "#VALUE!",
+                  address: imageUrl,
+                  altText: imageAltText
+                }
+              ]
+            ];
+
+            await context.sync();
+          });
+        }
+
+        async function retrieveImageInfo() {
+          await Excel.run(async (context) => {
+            // This method retrieves image data from a selected cell and displays it in the existing input fields.
+            
+            // Load the active cell information.
+            const activeCell = context.workbook.getActiveCell();
+            activeCell.load("valuesAsJson");
+            await context.sync();
+
+            // Get image data from the active cell.
+            const values = activeCell.valuesAsJson;
+            const webImageData = values[0][0] as Excel.WebImageCellValue;
+            const webImageUrl = webImageData.address;
+            const webImageAltText = webImageData.altText;
+
+            if (!webImageUrl) {
+              console.log("The selected cell is missing an image URL. Make sure to select a cell that contains an image.");
+              return;
+            }
+
+            // Assign image data to corresponding input fields.
+            $("#url").val(webImageUrl);
+            $("#alt-text").val(webImageAltText);
+          });
+        }
+
+        async function openImage() {
+          await Excel.run(async (context) => {
+            // This method retrieves the image URL from the selected cell and opens that image in a new browser tab.
+
+            // Load the active cell information.
+            const activeCell = context.workbook.getActiveCell();
+            activeCell.load("valuesAsJson");
+            await context.sync();
+
+            // Get image URL from the active cell.
+            const values = activeCell.valuesAsJson;
+            const webImageData = values[0][0] as Excel.WebImageCellValue;
+            const webImageUrl = webImageData.address;
+
+            if (!webImageUrl) {
+              console.log("The selected cell is missing an image URL. Make sure to select a cell that contains an image.");
```

Consider shortening this a little.
```suggestion
              console.log("The selected cell is missing an image URL. Select a cell that contains an image.");
```